isso/db/__init__.py

```diff
@@ -78,20 +78,32 @@ def migrate(self, to):
             bf = memoryview(Bloomfilter(iterable=["127.0.0.0"]).array)
 
             with sqlite3.connect(self.path) as con:
-                con.execute('UPDATE comments SET voters=?', (bf, ))
-                con.execute('PRAGMA user_version = 1')
-                logger.info("%i rows changed", con.total_changes)
+                con.execute("BEGIN TRANSACTION")
+                try:
+                    con.execute('UPDATE comments SET voters=?', (bf, ))
+                    con.execute('PRAGMA user_version = 1')
+                    con.execute("COMMIT")
+                    logger.info("Database migration to version 1 finished, %i rows changed", con.total_changes)
+                except sqlite3.Error as e:
+                    logger.error("Database migration to version 1 failed: %s", e)
```

Comment:
If a migration fails (which should _never_ happen except for corrupted or fiddled-with databases), should we bail (`sys.exit(1)` / `raise`) out entirely? `isso.db` will otherwise attempt to write to non-existent DB columns and fail, potentially losing comments.

Reply:
That's a good point, thank you.
I've added an exception for such cases.

Reply:
Great, thanks.

Just FYI, emitting a `logger.error` and then throwing a `RuntimeError` with the same message is a bit superfluous, I'll fix that in a follow-up commit.